evm/src/executor/inspector/cheatcodes/mod.rs

```diff
@@ -269,7 +269,7 @@ where
                     Return::Revert,
                     remaining_gas,
                     format!(
-                        "Expected a call to 0x{} with data {}, but got none",
+                        "Expected a call to {} with data {}, but got none",
```

Comment:
The display impl of `Address` continues to be a pain, sigh, I think we should print the whole address here?
```suggestion
                        "Expected a call to {:?} with data {}, but got none",
```